go-controller/pkg/util/node_annotations.go

```diff
@@ -51,6 +51,9 @@ const (
 	// OvnDefaultNetworkGateway captures L3 gateway config for default OVN network interface
 	ovnDefaultNetworkGateway = "default"
 
+	// ovnNodeManagementPort is the constant string representing the annotation key
+	ovnNodeManagementPort = "k8s.ovn.org/node-mgmt-port"
```

Comment:
I think it should be `dpu-host-mgmt-port`.

Reply:
What if we want to expose this information for full mode too?

Reply:
I'm not aware to which modes this annotation is applicable. Also changing it would affect existing clusters. Lets not touch it, since it is not a part of this PR

Reply:
I'm in favor of following the current scheme. The management port MAC address is already exposed using 

`k8s.ovn.org/node-mgmt-port-mac-address`

My proposed key for the management port itself is:

`k8s.ovn.org/node-mgmt-port`

Reply:
Ok